controller/Dockerfile.controller

```diff
@@ -1,136 +1,56 @@
-ARG builderImage="ghcr.io/microsoft/retina/retina-builder:0.0.1"
-ARG toolsImage="ghcr.io/microsoft/retina/retina-tools:0.0.1"
+# go binaries
+FROM --platform=$BUILDPLATFORM mcr.microsoft.com/oss/go/microsoft/golang:1.21 AS gobins
```

Comment:
do you want to bump this to 1.22 now or wait for [this ](https://github.com/microsoft/retina/pull/301)to go in?

Reply:
can bump it now, I am about to push one more change here anyway which collapses the win/linux dockerfiles together